app/Moderation/Moderations/Stick.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * @author    MyBB Group
+ * @version   2.0.0
+ * @package   mybb/core
+ * @license   http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+namespace MyBB\Core\Moderation\Moderations;
+
+use McCool\LaravelAutoPresenter\HasPresenter;
+use MyBB\Core\Moderation\ReversibleModerationInterface;
+use MyBB\Core\Moderation\SourceableInterface;
+use MyBB\Core\Presenters\Moderations\StickPresenter;
+
+class Stick implements ReversibleModerationInterface, HasPresenter, SourceableInterface
+{
+    /**
+     * @return string
+     */
+    public function getKey()
+    {
+        return 'stick';
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'Stick';
```

Comment:
Should this not use translations? The key should be English, but the name should be translatable I believe?